charts/quantumleap/README.MD

```diff
@@ -3,10 +3,12 @@
 This directory contains a Kubernetes chart to deploy a [Quantum Leap](https://github.com/smartsdk/ngsi-timeseries-api) cluster using Deployment.
 
 ## Prerequisites Details
-* Kubernetes 1.9+
-* Quantum Leap >= 0.7.6
+
+- Kubernetes 1.9+
+- Quantum Leap >= 0.7.6
 
 #### Note: Quantum Leap version
+
 This chart is **not** fully compatible with Quantum Leap versions earlier
 than `0.7.6`. However, it is still possible to use this chart with an older
```

Comment:
i think this needs to be changed to 0.8.0? did you test your chart change with 0.7.6?

Reply:
I've not tested it with 0.7.6 but i cannot see any reason why it won't work with 0.7.6.

As the only change I made are new envs (except for the removal of `CRATE_DEFAULTDB`, but this does not seem to be present in 0.7.6 either), a version of QL which does not know these envs will simply ignore them.

I've changed the default value for `config`, so for QL versions without the `QL_DEFAULT_DB` env you need to set the `config` variable to `{ tenants: {}, default-backend: "Crate" }`, as it was before, if you want to change the default backend.

Please correct me if I'm wrong

Reply:
I've made it clearly now in the readme, hope you agree